ip/draft-ietf-detnet-ip.xml

```diff
@@ -546,9 +555,9 @@ DetNet                    |L2/SbN|          |L2/SbN|
           One additional consideration for DetNet nodes which support CoS
           services is that they MUST ensure that the CoS service classes do
           not impact the congestion protection and latency control mechanisms
-          used to provide DetNet QoS.  This requirement is similar to
-          requirement for MPLS LSRs to that CoS LSPs do not impact the
-          resources allocated to TE LSPs via <xref target="RFC3473"/>.
+          used to provide DetNet QoS.  This requirement is similar to the 
+		  requirement for MPLS LSRs that CoS LSPs must not impact the resources 
```

Comment:
It's generally good practice to avoid lower cased conformance language wherever possible:
s/must/cannot

Reply:
OK. I have updated this.